flow/designs/asap7/jpeg_lvt/config.mk

```diff
@@ -0,0 +1,18 @@
+export PLATFORM               = asap7
+
+export DESIGN_NAME            = jpeg_encoder
+export DESIGN_NICKNAME        = jpeg_lvt
+
+export VERILOG_FILES          = $(sort $(wildcard ./designs/src/$(DESIGN_NICKNAME)/*.v))
+export VERILOG_INCLUDE_DIRS   = ./designs/src/$(DESIGN_NICKNAME)/include
+export SDC_FILE               = ./designs/$(PLATFORM)/$(DESIGN_NICKNAME)/jpeg_encoder15_7nm.sdc
+export ABC_AREA               = 1
+
+export CORE_UTILIZATION       = 30
+export CORE_ASPECT_RATIO      = 1
+export CORE_MARGIN            = 2
+export PLACE_DENSITY          = 0.60
+
+export DFF_LIB_FILE           = $($(CORNER)_DFF_LIB_FILE)
```

Comment:
This should be in the platform config.mk

Reply:
The DFF_LIB_FILE statement? It was already there in many files:

```
designs/asap7/ethmac/config.mk:export DFF_LIB_FILE           = $($(CORNER)_DFF_LIB_FILE)
designs/asap7/ibex/config.mk:export DFF_LIB_FILE           = $($(CORNER)_DFF_LIB_FILE)
designs/asap7/jpeg/config.mk:export DFF_LIB_FILE           = $($(CORNER)_DFF_LIB_FILE)
designs/asap7/jpeg_lvt/config.mk:export DFF_LIB_FILE           = $($(CORNER)_DFF_LIB_FILE)
designs/asap7/sha3/config.mk:export DFF_LIB_FILE           = $($(CORNER)_DFF_LIB_FILE)

```
Let me know if I should move it for all the files.

Reply:
I looked at it some more and this whole setup is screwy.  Let's leave it as is in this PR and I'll fix it for the platform afterwards.